DOCS/man/mpv.rst

```diff
@@ -270,6 +270,34 @@ Alt+2 (and Command+2 on macOS)
 Command + f (macOS only)
     Toggle fullscreen (see also ``--fs``).
 
+(The following keybindings open a selector in the console that lets you choose
+from a list of items by typing part of the desired item and/or by navigating
+them with keybindings: ``Down`` and ``Ctrl+n`` go down, ``Up`` and ``Ctrl+p`` go
+up, ``Page down`` and ``Ctrl+f`` scroll down one page, and ``Page up`` and
+``Ctrl+b`` scroll up one page.)
+
+g-p
+    Select a playlist entry.
+
+g-t
+    Select a track.
```

Comment:
Maybe add also selectors for specific types of tracks? Maybe little bit bloat, but people probably never want to select video track.

Also I got an idea to output audio device selection. Would it be possible to add, if we are adding everything...

Reply:
Sure I've also been considering doing it for every track type.

Output audio device should be possible too.

Reply:
Added.